src/utils/sqlite.jl

```diff
@@ -37,3 +37,22 @@ function backup(
         end
     end
 end
+
+"""
+Wrapper around SQLite.DBInterface.execute to provide debug log messages.
+"""
+function execute(db::SQLite.DB, query::AbstractString, log_group::Symbol)
+    @debug "Execute SQL" _group = log_group query
+    return SQLite.DBInterface.execute(db, query)
+end
+
+"""
+Run a query to find a count. The query must produce a column called count with one row.
```

Comment:
Current behavior seems to ignore rows 2 and up, should this be an error instead?

Reply:
We had some related conversation in the last PR, but sure, we can assert this. The use case is that the user has has asked for a count and there can only be one row.

Reply:
Nevermind. The SQLite.jl people don't seem to have fully implemented the Tables interface.
```
MethodError: no method matching length(::SQLite.Query{false})
```
I'm confident that this is not a problem.

Reply:
Okay